index.php

```diff
@@ -0,0 +1,44 @@
+<?php
+    $db = new PDO('mysql:host=db; dbname=collection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $query = $db->prepare("SELECT * FROM `playing_cards`;");
+    $query->execute();
+    $cardSets = $query->fetchAll();
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Playing Card Collection</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<header>
+    <h1>My Playing Cards</h1>
+</header>
+<main>
+    <div class="container grid">
+        <?php
+            foreach ($cardSets as $cardSet) {
```

Comment:
Would be worth some kind of check here incase there are no cards in the collection (a future story is deleting cards) - just display an error message like 'there are no cards in the collection' in that case